redwood-lazylayout-dom/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'org.jetbrains.kotlin.js'
+
+redwoodBuild {
+}
```

Comment:
Deliberately not `publishing()` this, as the current implementation doesn't properly implement lazy loading in the DOM. I wanted to make sure we don't accidentally publish a broken version of this. We should enable publishing once #1048 has been completed.